src/components/SpiceList.js

```diff
@@ -1,16 +1,15 @@
 import React from 'react'
-// TODO: import the SpiceItem component here
+import SpiceItem from "./SpiceItem"
 
-// class component
 class SpiceList extends React.Component {
 
-  renderSpices() {
-    // TODO: update this function to return an array of SpiceItem components
-    // If you get an error message about a 'key' prop, make sure to fix it before submitting your code!
+  renderSpices = () => {
+    return this.props.spices.map(spice => <SpiceItem spice={spice} />)
```

Comment:
Make sure to use the `key` prop any time you're returning an array of JSX, like this:

```jsx
this.props.spices.map(spice => <SpiceItem key={spice.id} spice={spice} />)
```